### ISSUE TITLE
policies are not running

### ISSUE THREAD

**Issue by yevgenypats** [cloudquery/cloudquery#617]
### Describe the Bug

This relates only to `dev` branch that should be merged soon.

Currently running the following

```
cloudquery fetch
cloudquery policy run gcp
```
produces the following error:

```
Anonymous telemetry collection and crash reporting enabled. Run with --no-telemetry to disable, or check docs at https://docs.cloudquery.io/docs/cli/telemetry
Initializing CloudQuery Providers...

✓ cq-provider-gcp@v0.7.0 verified    1s  100 %

Finished provider initialization...

Checking available provider updates...

Starting policies run...

✓ policy "gcp" -  evaluating -     0s   Finished Checks: 83/83

❌ Failed to run policies: gcp: provider gcp version >= 0.4.0 is not defined in configuration.

gcp: provider gcp version >= 0.4.0 is not defined in configuration

```

Even though policy is on latest version

Same also occures when fetching latest aws and running policy

```
yevgenyp@yevgenys-mbp cloudquery % ./cloudquery policy run aws
Anonymous telemetry collection and crash reporting enabled. Run with --no-telemetry to disable, or check docs at https://docs.cloudquery.io/docs/cli/telemetry
Initializing CloudQuery Providers...

✓ cq-provider-aws@v0.11.1 verified    1s  100 %

Finished provider initialization...

Checking available provider updates...

Starting policies run...

✓ policy "aws" -  evaluating -     0s   Finished Checks: 221/221

❌ Failed to run policies: aws: provider aws version >= 0.10.0 is not defined in configuration.

aws: provider aws version >= 0.10.0 is not defined in configuration
```

**Comment by roneli** [comment]
You require the provider to defined in the config

**Comment by roneli** [comment]
fixed #615
